lib/listen/adapter/darwin.rb

```diff
@@ -62,22 +62,22 @@ def _process_changes(dirs)
       end
 
       def _process_event(dir, path)
-        _log(:debug) { "fsevent: processing path: #{path.inspect}" }
+        Listen.logger.debug { "fsevent: processing path: #{path.inspect}" }
         # TODO: does this preserve symlinks?
         rel_path = path.relative_path_from(dir).to_s
         _queue_change(:dir, dir, rel_path, recursive: true)
       end
 
       def _run_worker(worker)
```

Comment:
Are we still using this method?

Reply:
Yes, it looks to me like it's called from the base class:
https://github.com/Invoca/listen/blob/de1e7cb808226ecfaa9bc3dc3a002812e8329fff/lib/listen/adapter/base.rb#L45

Reply:
I think @ioquatix was referring to `_run_worker`, rather than `_process_event`.  I don't think `_run_worker` is used elsewhere, at least according to `git grep _run_worker`.

Reply:
Oh, thanks for spotting my misunderstanding @jonathanhefner. That is indeed true. This unused `_run_worker` might have been a from a mistaken merge conflict resolution. Deleting it now.